### ISSUE TITLE
Reader: Tags light refresh

### ISSUE THREAD

**Issue by jasmussen** [Automattic/wp-calypso#100360]
Tags page:

<img src="https://uploads.linear.app/f629fcde-6f9b-48e8-8af3-60eed8823bff/e9932500-039f-4144-bc97-c8cb4b5a84ff/c90be1fd-2ae0-42d7-affd-f4148bcfe87c?signature=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJwYXRoIjoiL2Y2MjlmY2RlLTZmOWItNDhlOC04YWYzLTYwZWVkODgyM2JmZi9lOTkzMjUwMC0wMzlmLTQxNDQtYmM5Ny1jOGNiNGI1YTg0ZmYvYzkwYmUxZmQtMmFlMC00MmQ3LWFmZmQtZjQxNDhiY2ZlODdjIiwiaWF0IjoxNzQ3MTI5NzQwLCJleHAiOjMzMzE3Njg5NzQwfQ.0g0IxeoT9KFTNOFjuyg4yuw_nVjGovDNXkW5iBlHS60 " alt="Image" width="1920" />

A few tasks to improve this page:

* Related to [#100349](https://github.com/Automattic/wp-calypso/issues/100349), the back button is not in a consistent placement.
* [DOTCON-46](https://linear.app/a8c/issue/7f320dd4-9dda-42f6-986a-fa93d63ea028)
* The "Add a tag" placeholder text does not meet contrast ratios
* The active state for the current chosen tag lacks state indication (such as bold text, or a background)
* The segmented control isn't used in this style elsewhere (See [#100357](https://github.com/Automattic/wp-calypso/issues/100357))
* The sidebar and stats in the title could use a positioning grid
* The title is poorly elided.

The page could use an overall design pass. Suggestions:

* Use standard core components for Add a tag, input, and a primary button for "Add".
* Use semibold for the active state.
* Don't elide the title. If it spans 5 lines, let it, maybe it's an aside type post.
